.github/dependabot.yml

```diff
@@ -16,9 +16,9 @@ updates:
       timezone: "Europe/Zurich"
     open-pull-requests-limit: 10
     ignore:
-      - dependency-name: django
+      - dependency-name: python-semantic-release
         versions:
-          - ">=4"
+          - ">=7"
```

Comment:
Should we add something like this?

```
  ignore:
  - dependency-name: django
    versions:
    - ">=4.3"
  ```

Reply:
No deprecations, so it should be ok to have the django 5 update queued by dependabot and merge when ready.

Reply:
@Yelinz I strongly disagree, we always want LTS versions..

Reply:
added ignore